### PR TITLE
fixed sword drop enchant bug

### PR DESCRIPTION
**Pull request by ImYenil**
Fixed sword drop enchant bug

### CODE REVIEW

**Comment by tomkeuper** [comment]
what bug?

**Comment by ImYenil** [comment]
> what bug?

https://github.com/tomkeuper/BedWars2023/issues/116

File: bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/team/BedWarsTeam.java [hunk 1]
```diff
@@ -219,6 +219,16 @@ public void sendDefaultInventory(Player p, boolean clean) {
...
+                    if (!getSwordsEnchantments().isEmpty()) {
+                        if (im == null) continue;
+                        if (nms.isSword(i)) {
+                            for (TeamEnchant e : getSwordsEnchantments()) {
+                                im.addEnchant(e.getEnchantment(), e.getAmplifier(), true);
+                            }
+                            i.setItemMeta(im);
+                        }
+                        p.updateInventory();
+                    }
```
Comment (tomkeuper) [inline bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/team/BedWarsTeam.java:231]: This wont work. You are in the sendDefaultInventory function which is only triggered when respawning or at the first spawn

Comment (ImYenil) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/team/BedWarsTeam.java:231]: > This wont work. You are in the sendDefaultInventory function which is only triggered when respawning or at the first spawn

I tested. it works

Comment (tomkeuper) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/team/BedWarsTeam.java:231]: I can guarantee you this doesnt work

Comment (tomkeuper) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/team/BedWarsTeam.java:231]: sendDefaultInventory is only called in the firstSpawn function and within the respawn member function

**Pull request closed without merging**